templates/scheduler/flux/variables.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
+  gtl_flag: ''  # to be overwritten by tests that need GTL
+  rocm_arch: 'gfx90a'
```

Comment:
This is a good example of something where we'd want to split `variables.yaml` into multiple files: this is probably the same across systems and is also not scheduler-specific.